src/home.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable import/no-cycle */
+
+import cheImage from './chef.jpg';
+import loadMenu from './menu';
+import LoadContact from './contact';
+
+const loadHome = () => {
+  const container = document.getElementById('content');
+  container.innerHTML = '';
+  container.innerHTML = `<nav id='tab'>
+  <ul>
+  <li id="home">HOME</li>
+  <li id="menu">MENU</li>
+  <li id="contact">CONTACT</li>
+</ul>
+</nav>
+
+<main>
+<h1>Welcome to Certil Restaurant</h1>
+<img id="chef" alt="chef" />
+<p>
```

Comment:
I would also recommend keeping your code [`DRY`](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself). 
This line seems to be repeated in all your modules:
```js
  const container = document.getElementById('content');
  container.innerHTML = '';
  container.innerHTML = `<nav>
  <ul>
  <li id="home">HOME</li>
  <li id="menu">MENU</li>
  <li id="contact">CONTACT</li>
  </ul>
  </nav>
  
  <main>
```

This means some section can be a constant imported to all module or a reusable module with a different parameter for different views.